modules/oe_editorial_corporate_workflow/oe_editorial_corporate_workflow.module

```diff
@@ -15,14 +15,40 @@ use Drupal\node\NodeInterface;
  */
 function oe_editorial_corporate_workflow_menu_local_tasks_alter(&$data): void {
   $node = \Drupal::routeMatch()->getParameter('node');
-  if ($node instanceof NodeInterface && !empty($data['tabs'][0]['entity.node.edit_form']['#link'])) {
+  if (!$node instanceof NodeInterface) {
+    return;
+  }
+
+  // Edit tab.
+  if (!empty($data['tabs'][0]['entity.node.edit_form']['#link'])) {
     if (_oe_editorial_corporate_workflow_is_last_revision_published($node)) {
       $data['tabs'][0]['entity.node.edit_form']['#link']['title'] = t('New draft');
     }
     else {
       $data['tabs'][0]['entity.node.edit_form']['#link']['title'] = t('Edit draft');
     }
   }
+
+  // Latest revision tab.
+  if (!empty($data['tabs'][0]['content_moderation.workflows:node.latest_version_tab']) &&
+    $data['tabs'][0]['content_moderation.workflows:node.latest_version_tab']['#active'] === TRUE
+  ) {
+    $data['tabs'][0]['content_moderation.workflows:node.latest_version_tab']['#link']['title'] = t('View draft');
+  }
+
+  // View tab.
+  if (!empty($data['tabs'][0]['entity.node.canonical']['#link'])) {
+    /** @var \Drupal\content_moderation\ModerationInformationInterface $moderation_info */
+    $moderation_info = \Drupal::service('content_moderation.moderation_information');
+    if ($moderation_info->isModeratedEntity($node) &&
```

Comment:
We have a problem here: this if will set title to "View draft" if `$moderation_info->isModeratedEntity($node)` is `FALSE`, which is something we don't want.